cpp/build-support/update-flatbuffers.sh

```diff
@@ -18,30 +18,22 @@
 # under the License.
 #
 
-# Run this from cpp/ directory. flatc is expected to be in your path
+# Run this from cpp/ directory. flatc is expected to be in your path.
+# The output directory can be passed as a positional argumenet,
+# it defaults to /cpp/src/generated.
 
 set -euo pipefail
 
 CWD="$(cd "$(dirname "${BASH_SOURCE[0]:-$0}")" && pwd)"
 SOURCE_DIR="$CWD/../src"
-PYTHON_SOURCE_DIR="$CWD/../../python"
 FORMAT_DIR="$CWD/../../format"
-TOP="$FORMAT_DIR/.."
 FLATC="flatc"
```

Comment:
How about running `flatc` from CMake directly instead of using this shell script?

Because:
* We need to use `flatbuffers::flatc` instead of `flatc`.
* If we use shell script, we can't use system FlatBuffers on Windows.